kadena-signing-api/default.nix

```diff
@@ -1,32 +1,22 @@
-{ pactRef ? "a6c5315682d8e2a9a047a3d903d7641df066bbda"
-, pactSha ? "0a000lxgsj3kjkmgm2mkfnx609hqa4gcmpw40fbwxyaxnp93sb1f"
+{ pactRef ? "ace18a2489bf86595af09316bd1c09794c326b77"
```

Comment:
It looks like these are no longer being used?  Also lines 7-10?  If it's being used for the pact dependency, shouldn't that come from kpkgs?